hack/.shellcheck_failures

```diff
@@ -148,7 +148,7 @@
 ./test/images/volume/rbd/create_block.sh
 ./test/images/volume/rbd/mon.sh
 ./test/images/volume/rbd/osd.sh
-./test/kubemark/common/util.sh
+./test/kubemark/configure-kubectl.sh
```

Comment:
this diff is weird, what happened here?

Reply:
ah removed in the next commit

Reply:
seems like the next commit should get squashed into this one, as otherwise it's a no-op?

Reply:
That likely occurred while I was rebasing the first time around, I'll go through and make sure everything is good.